orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/util/ContextParameterProcessor.java

```diff
@@ -67,15 +68,18 @@ public ContextParameterProcessor() {
                 new UserConfiguredUrlRestrictions.Builder().build(),
                 new HttpClientUtils(new UserConfiguredUrlRestrictions.Builder().build()))),
         new DefaultPluginManager(),
-        DynamicConfigService.NOOP);
+        DynamicConfigService.NOOP,
+        new ExpressionProperties());
```

Comment:
I realize it's off-topic for this PR, but it would be lovely for this constructor to disappear / move into some test class somewhere.  I do see a bunch of different test classes using this, so this is idle chatter...still though, it's a bummer to have test code polluting what we ship.